bodhi/server/models.py

```diff
@@ -923,6 +933,7 @@ class Update(Base):
     status = Column(UpdateStatus.db_type(),
                     default=UpdateStatus.pending,
                     nullable=False)
+    ci_status = Column(UpdateCiStatus.db_type(), default=None, nullable=True)
```

Comment:
Since it doesn't do anything I don't have a particularly strong opinion one way or the other, but I think marking a column as nullable is explicit since it indicates that the column itself allows null values which _must_ mean the mapped Python object's attribute is allowed to be null and if you don't provide a value... Well, it's going to be null.